Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.h

```diff
@@ -85,8 +85,8 @@ ALLOW_NEW_API_WITHOUT_GUARDS_END
     void removeVideoTrack(VideoTrackPrivate&);
     void removeTextTrack(InbandTextTrackPrivate&);
 
-    MediaPlayer::NetworkState networkState() const override;
-    MediaPlayer::ReadyState readyState() const override;
+    MediaPlayer::NetworkState networkState() const final;
+    MediaPlayer::ReadyState readyState() const final;
```

Comment:
These unrelated `override` to `final` changes are a mistake:
- they make it harder to pick out and review the changes necessary for this fix
- they will make future `blame` analysis more complicated
- I believe it is contrary to standard WebKit practice

Reply:
done